RevenueCatUI/UIKit/PaywallViewController.swift

```diff
@@ -82,12 +86,24 @@ public class PaywallViewController: UIViewController {
         self.addChild(self.hostingController)
         self.view.addSubview(self.hostingController.view)
         self.hostingController.didMove(toParent: self)
-    }
-
-    public override func viewWillLayoutSubviews() {
-        super.viewWillLayoutSubviews()
+        hostingController.view.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Explicit self here and below:
```suggestion
        self.hostingController.view.translatesAutoresizingMaskIntoConstraints = false
```